### DISCUSSION TITLE
Todavia no veo el grupo en MS Teams

### DISCUSSION THREAD

**Discussion by KanoB** [Launch-X-Latam/MisionFrontEnd#432] [Team Posts]
Hola a todos!

Empece un poco tarde al curso y no se si se deba a eso, pero no puedo ver todavia ningun grupo en el MS Teams.
Complete todo el proceso de onboarding, y he estado revisando el material de la mision de front end, sin embargo me quedo la duda de si hay sesiones en vivo.

Gracias de antemano.

**Comment by gumodi35** [comment]
Si hay sesiones en vivo ** lunes y miércoles 7:00pm hora México**, aún faltaban algunos accesos seguro te estarán llegando en breve.... Las **seciones se quedan grabadas** para que las veas cuando te sea posible. Hay categorias para hacer **preguntas y respuestas**, quizá está pregunta no esté en el issue adecuado, leete bien el **onboarding** y ve si tienes acceso a las secciones del **teams**, cualquier cosa manda correo a **contacto innovación** o deja tu mensaje en discusiones. Igual si tienes dudas manda **DM* *

**Comment by KanoB** [comment]
Gracias @gumodi35 
Por lo que lei en el mail group de Mision FrontEnd hay un detalle todavia de accesos para el MS Teams.
Espero pronto poder entrar.

Saludos, K

**Comment by Cdmonytoua1** [comment]
Hola, al parecer me encuentro en la misma situación , aun no puedo entrar al grupo de MS TEAMS

**Comment by 3nriquecastro** [comment]
yo me encuentro en la misma posición en mi primera instancia le escribí al correo de ayuda y me dijeron que haga lo onboarding,y tampoco me metieron a ningún grupo de teams

**Comment by taniagd** [comment]
En el onboarding vienen todas las instrucciones para añadirse a teams. Una vez que manden su solicitud y alguien les de acceso, podrán visualizar todo el contenido.

**Comment by Yhor93** [comment]
Yo estoy igual que uds aún no he ingresado a un grupo, alguno tiene
respuesta

**Comment by DanielChavezZ** [comment]
En el onboarding vienen todas las instrucciones paso por paso para entrar a teams

**Comment by 3nriquecastro** [comment]
pero nos manda a una google forms que ya no funciona

**Comment by INVALIDO12** [comment]
Yo yo mande todo y no veo que me metan

**Comment by Annamalag** [comment]
Hola chicos.! Yo tampoco podía ingresar y mandé msjs al instagram de
innovacción virtual y ellos me solucionaron el problema en un par de
horas.. saludos

**Comment by gumodi35** [comment]
En teoría les debería de funcionar, luego de rellenar los datos con su repositorio de GitHub y demás. Intente lo nuevamente, al enviar el formulario les debería llegar la invitación al correo que registraron desde el inicio.

**Comment by KanoB** [comment]
Ya me dieron acceso al grupo de Teams.
Ayer volví a hacer click en el correo con mi invitación de QR. Creo que ha
estado saturado el servicio. Así que repetirlo ayer hizo que vieran mi
petición de acceso y me pudiera. aceptar.

Recomiendo repitan ese paso.

Saludos, K

**Comment by jandergomez10** [comment]
Hola amigos!! Envíen un mensaje al instagram https://instagram.com/innovaccionvirtual?igshid=YmMyMTA2M2Y=

Yo envié ahí y me solucionaron muy rápido

Saludos

Jander Gomez
________________________________

**Comment by lightshadowcm** [comment]
ya lo envie espero que me puedan ayudar

**Comment by gumodi35** [comment]
Luego se satura el servidor, trata de hacerlo nuevamente.

**Comment by DuwanParra** [comment]
alguien sabe cómo accedo al material que dicen que hay aquí ?

**Comment by gumodi35** [comment]
El material esta en el repositorio 🔖 de Mision frontend, hay puedes checar.

**Comment by alexander-ale** [comment]
hola buen día, tengo un problema. No me deja restablecer la contraseña, se
me olvido la contraseña para ingresar al team.

**Comment by gumodi35** [comment]
intenta nuevamente verificar tu cuenta o manda mensaje a innovaccion.

**Comment by craonay** [comment]
Solo hay clases en vivo 2 días a la semana?

**Comment by Isaiasmj** [comment]
si, miercoles y viernes las clases

**Comment by craonay** [comment]
En la semana mientras tanto vamos revisando el material de estudio fronted y lo complementamos con las clases ¿cierto?

**Comment by Isaiasmj** [comment]
asi es, amigo

**Comment by Dany-Molina** [comment]
Hola que tal una consulta en donde encuentro el onborxing para realizarlo que ya busque y no lo encuentro

**Comment by gumodi35** [comment]
esta en el repositorio de la organización!

**Comment by Hiddencelc** [comment]
hola a todos, también recien pudiendo entrar y validar el correo, con muchas ganas de aprender

**Comment by Illescas970103** [comment]
Hola a todos me voy integrando a estas sesiones, de igual forma ya solicite acceso al grupo de teams, saben si tardan en aceptarlo?

**Comment by Scetho** [comment]
hoy hubo sesion?

**Comment by ivansike** [comment]
Es el miércoles :) 

Enviado desde mi iPhone

**Comment by ivonama** [comment]
Las sesiones se quedaran grabadas ??

**Comment by Reyesoy** [comment]
Sí quedan grabadas, Sherpa publicó en Teams, "Grabaciones del primer módulo", y ahí aparecen los links de las sesiones. 
Saludos

**Comment by GabRafaDeveloper** [comment]
buenas tarde, aun no me aceptan en el grupo de TEAM de launch x
mi correo es lx14307@innovaccion.mx

**Comment by EdwinSHz** [comment]
Tengo el problema con TEAMS, no me han aceptado la solicitud, mi correo es: ***@***.***


Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by GabRafaDeveloper** [comment]
> Sí quedan grabadas, Sherpa publicó en Teams, "Grabaciones del primer módulo", y ahí aparecen los links de las sesiones.
> Saludos
buenas tarde, aun no me aceptan en el grupo de TEAM de launch x
mi correo es [lx14307@innovaccion.mx](mailto:lx14307@innovaccion.mx)

**Comment by jandergomez10** [comment]
Hola amigos, las personas que tienen problemas escriban al instagram  https://instagram.com/innovaccionvirtual?igshid=YmMyMTA2M2Y=
A mi me solucionaron muy rápido

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by anni365** [comment]
Hola! Los que aún tengan inconvenientes para entrar, pueden escribir a la cuenta de Instagram de Innovacción.
Yo lo hice y lo solucionaron muy rápido

**Comment by andresbenitez12** [comment]
Asegurate de haber ingresado al Teams institucional con el correo de Innovacción.

**Comment by jhonnatan971** [comment]
Buen dia, el inconveniente que tengo es que no me aparece ningún contenido
dentro del teams, está vacío.
Ya escribí al instagram, pero no he obtenido respuesta
Espero me puedan ayudar, muchas gracias  .

**Comment by AldairNava** [comment]
En el onbording en gitbuh está un enlace para solicitar que te envíen la invitación

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by lightshadowcm** [comment]
te recomiendo ver este tiktok https://vm.tiktok.com/ZMFFTwoxo/

**Comment by ivonama** [comment]
Hola, buen día no puedo acceder a la reunión de teams, con el pase les pasa lo mismo ??

**Comment by Jorge260399** [comment]
Hola, que tal, me pasa que igual me ha llegado el correo con el QR para entrar a MS Teams, pero solo me sale lo de solicitud enviada, hay que esperar a que te acepten?

**Comment by Laoset** [comment]
Me pasa lo mismo que a vos Jorge, supongo que habrá que esperar. Saludos.

**Comment by kanob22** [comment]
Si Jorge con ese QR se vuelve a enviar tu solicitud de ingreso al grupo de teams. Deberás solo esperar un rato. Conmigo tardaron 1 día aproximadamente. Saludos,K

Enviado desde Outlook para iOS<https://aka.ms/o0ukef>
________________________________
De: Jorge260399 ***@***.***>
Enviado: Thursday, October 6, 2022 9:11:59 PM
CC: Subscribed ***@***.***>
Asunto: Re: [Launch-X-Latam/mision-frontend] Todavia no veo el grupo en MS Teams (#1)


Hola, que tal, me pasa que igual me ha llegado el correo con el QR para entrar a MS Teams, pero solo me sale lo de solicitud enviada, hay que esperar a que te acepten?

**Comment by AranxaMo** [comment]
A mi me pasa lo mismo, solo me aparece que la solicitud fue enviada. Esperare al día de mañana para ver si me aceptan :)

**Comment by HansonAlejandro** [comment]
Solo nos queda esperar, a que nos autoricen, Saludos compañeros

**Comment by TwinDmedina** [comment]
Mismo problema aqui

**Comment by RafaelTorres5** [comment]
Igual mente

**Comment by MarcosQuintana10** [comment]
Hola a todos, me sucede lo mismo, pero veo que no soy el unico. Bueno solo toca esperar :D

**Comment by Mangelzz** [comment]
Me pasa lo mismo, entré un poco tarde y ando esperando que me den acceso al grupo mision frontend en teams, esperemos :(

**Comment by Alejandra0831** [comment]
Hola chicos, ya  hoy es la cuarta clase que tendremos.... les sugiero escribir por Instagram para que les den solución del acceso (yo lo  hice así y en cuestión de unas horas ya tenía acceso a Teams), El tiempo apremia y las asesorías son los miercoles y los viernes

Les sugiero por ahora:

1. Leer los temas del repositorio https://github.com/Launch-X-Latam/MisionFrontEnd, en las clases con el Mission Comander hemos visto el primer módulo (https://github.com/romarpla/cursoFrontEnd/blob/main/01%20-%20INTRO/README.md)
2. Aun no hemos hecho la práctica de ese módulo sin embargo se han tratado, como les digo varios temas importantes.
3. Para que no lleguen muy crudos en lo que llevamos, es importante echarle un vistazo a las 3 clases pasadas (la del viernes 28 fue un laboratorio)

Estos son los contactos 

Instagram: https://www.instagram.com/innovaccionvirtual/

Correo: contacto@innovaccion.mx

Un abrazo y ..."Vamos que vamos!!!!"

**Comment by lucia123lu** [comment]
Hola, buenas, quería consultar cual es el grupo en teams que hay que acceder, porque directamente no me aparece ninguno en equipos y estoy bastante perdida, gracias

________________________________
De: Miguel ***@***.***>
Enviado: viernes, 7 de octubre de 2022 10:52
Cc: Subscribed ***@***.***>
Asunto: Re: [Launch-X-Latam/mision-frontend] Todavia no veo el grupo en MS Teams (#1)


Me pasa lo mismo, entré un poco tarde y ando esperando que me den acceso al grupo mision frontend en teams, esperemos :(

**Comment by Edwards28** [comment]
Hola a todos, ¿si están revisando que ese QR les llegue a su cuenta de Outlook, verdad?
Porque ahí es que caen y no a su cuenta de Gmail sino que a la cuenta que crearon con los datos de acceso que nos dieron.

Espero puedan entrar 😄

**Comment by lucia123lu** [comment]
hola, no encontré ningún QR en outlook la verdad, no se si es que no llego o que no lo encuentro
________________________________
De: Edwards28 ***@***.***>
Enviado: viernes, 7 de octubre de 2022 13:29
Cc: Lucía Stugelmayer ***@***.***>; Comment ***@***.***>
Asunto: Re: [Launch-X-Latam/mision-frontend] Todavia no veo el grupo en MS Teams (#1)


Hola a todos, ¿si están revisando que ese QR les llegue a su cuenta de Outlook, verdad?
Porque ahí es que caen y no a su cuenta de Gmail sino que a la cuenta que crearon con los datos de acceso que nos dieron.

Espero puedan entrar 😄

**Comment by lightshadowcm** [comment]
creo que es juntarnos en algun momento y hacer una reunión y expicarlo de
manera por meet, también recomiendo este tiktok
https://vm.tiktok.com/ZMFYVugU3/

**Comment by plopez10** [comment]
Hola, 

Recibi el correo con los datos hasta el dia de ayer, solamente estoy aprobacion para entrar al equipo en Teams :)

**Comment by lightshadowcm** [comment]
yo fui al instagram presente mi problema con mi nombre para que la señorita
zoe me añadio

**Comment by AymaraMolina** [comment]
No puedo ingresar al correo electrónico, me genera un error al hacer el cambio de contraseña. Y por esto mismo no he podido ingresar a teams

**Comment by Edwards28** [comment]
> 

Creo que también te dio el mismo error que a mí, lo que hice fue volver a intentarlo y me dejó crear la nueva contraseña

**Comment by alexander-ale** [comment]
![c](https://user-images.githubusercontent.com/12107094/194630481-4901e084-1ec0-4bac-a022-0306cac09103.jpg
Se me olvido la contraseña, lo tenia guardado automanticamente google. Me pueden dar otra contraseña. El correo de la cuenta teams es lx06501@innovaccion.mx

**Comment by AymaraMolina** [comment]
Ya intente varias veces y no funciona me muestra un error de autenticación.

**Comment by gumodi35** [comment]
Hola !

1.- Cambiaste la contraseña la primera vez que accediste?
2.- Intenta limpiar el historial de navegacion y vuelve a iniciar.
3.- En caso de que olvidaste tu contraseña, trata de acceder al metodo de recuperacion.
4.- Si ningun metodo te funciona, manda un email a contacto innovaccion y espera respuesta, a veces tarda un poco.

La primera vez que accedes al grupo de teams, no vas a ver el grupo de Launch X LATAM eso es lo mas normal, espera un poco para que tengas el acceso, por ejemplo si hoy logras acceder al grupo de teams usando tu cuenta de Launch X,  lo mas normal es que al dia siguiente logres tener el acceso, las seciones quedan grabadas y tienes un lapso de 24 a 48 hrs. para ver la sesion. 
Cualquier duda puedes colocar en la seccion de Q&A para que la comunidad te pueda apoyar. Saludos!!
